packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/android/signin.md

```diff
@@ -1,7 +1,36 @@
-First initialize the Okta AppAuth SDK in the `Activity#onCreate` method of the Activity that you are using to sign users in to your app. In this example, we call it LoginActivity:
+First initialize the Okta OIDC SDK in the `Activity#onCreate` method of the Activity that you are using to sign users in to your app. In this example, we call it LoginActivity:
 
-code example
+```java
+public class LoginActivity extends AppCompatActivity {
+    private OIDCAccount account;
+    private AuthenticateClient client;
 
-After the `OktaAppAuth` instance is initialized, you can start the authorization flow by simply calling `login` whenever you're ready:
+    account = new OIDCAccount.Builder()
+        .withResId(this, R.id.okta_oidc_config)
+        .create();
 
-code example
\ No newline at end of file
+    client = new AuthenticateClient.Builder()
+        .withAccount(account)
+        .withContext(this)
+        .withStorage(new SimpleOktaStorage(this))
+        .withTabColor(getColorCompat(R.color.colorPrimary))
+        .create();
+}
+```
+
+After the `AuthenticateClient` instance is initialized, you can start the authorization flow by simply calling `logIn` whenever you're ready:
+
+```java
+client.logIn(this, null);
+```
+
+If you need to add extra request parameters to the [authorize](https://developer.okta.com/docs/api/resources/oidc/#authorize) endpoint, `AuthenticationPayload` can be used:
```

Comment:
If you need to add extra request parameters to the [authorize](https://developer.okta.com/docs/api/resources/oidc/#authorize) endpoint, you can use `AuthenticationPayload`: